### DISCUSSION TITLE
New Item Pipe models

### DISCUSSION THREAD

**Discussion by Rearth** [Rearth/Oritech#16] [Todos]
The current item pipes were trying to imitate the chest color scheme to make it clear that they operate on items. However, they don't look good. A better model (or at least an idea for one) is needed

**Comment by jshipley** [comment]
I think that the idea is fine (Tom's Simple Storage does pretty much the same thing).

I see a few problems with the implementation of it, though:
* The texture looks a little too busy/noisy
* The corners without any border look a little weird. Giving them a metallic looking corner (like the fluid pipes have) might look better.
* The bright red and green particles when you break the pipe seem out of place
* The gray edge has a bright spot at the connection point between pipes
* The pipe is made with wood and nickel. It might be nice if there was a brighter nickel color on the edge instead of the gray

I'll try to do a texture tomorrow.

**Comment by jshipley** [comment]
![2024-07-31_16 21 08](https://github.com/user-attachments/assets/1f924549-39bc-4f70-bc94-6666c7c812ac)

Here's my attempt. Trying to use nickel colors really didn't work well, because the palette was too similar to the chest palette. I ended up using some of the orange and black colors from other Oritech machine blocks.

Here's the pipe texture in a resource pack if you want to take a closer look (I also changed the uv coordinates for the models and removed a lot of the unused stuff in the spritesheet).

[item_pipe_retexture.zip](https://github.com/user-attachments/files/16448484/item_pipe_retexture.zip)

Let me know if you want anything changed. If you like it, I can send a PR.

**Reply by Rearth** [reply]
Wow those already look much better than what I currently have. One small addition for the extraction interface might be a small arrow head pointing outward to further indicate that the pipe is set to export.
But even with this state it's already a huge improvement, a PR for this would be hugely appreciated

**Comment by Rearth** [comment]
Love the pipes from your PR. I'll close this discussion for now, as I think that the issue has been solved
